arm/datalake-store/filesystem/version.go

```diff
@@ -24,7 +24,7 @@ import (
 
```

Comment:
datalake-store catalog and filesystem are not an management service. It should not exists in the ARM namespace.